tools/scripts/changelog_test.rb

```diff
@@ -0,0 +1,51 @@
+#! /usr/bin/ruby
```

Comment:
@juliogonzalez  this is how easy and understable is now the changelog test .

All the trigger logic removed and it is  done by the engine.

I am not finish, but this is a great improvement to isolate this test from the engine.